EIPS/eip-5022.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 5022
+title: Increase price of SSTORE from zero to non-zero to 40k gas
+author: Green (@greenlucid)
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-04-20
+discussions-to: https://ethereum-magicians.org/t/eip-proposal-increase-cost-of-sstore-from-20k-to-x-when-creating-new-storage/7614
+---
+
+### Abstract
+
+Increase the price of the SSTORE opcode from `20_000` gas to `40_000` gas when the original slot is zero and the resultant slot is non-zero.
```

Comment:
```suggestion
Increase the price of the SSTORE (0x55) opcode from `20_000` gas to `40_000` gas when the original slot is zero and the resultant slot is non-zero.
```
Opcodes should be referenced at least the first time in the document in the form:
> REVERT (0xfe)